notification/slack/channel.go

```diff
@@ -44,15 +45,17 @@ func NewChannelSender(ctx context.Context, cfg Config) (*ChannelSender, error) {
 		chanTht: newThrottle(time.Minute / 50),
 		listTht: newThrottle(time.Minute / 50),
 
-		listCache: newTTLCache(250, time.Minute),
-		chanCache: newTTLCache(1000, 15*time.Minute),
+		listCache:   newTTLCache(250, time.Minute),
+		chanCache:   newTTLCache(1000, 15*time.Minute),
+		teamIDCache: newTTLCache(1, 5*time.Minute),
```

Comment:
This should probably just be cached based on the access token value, rather than time